package.json

```diff
@@ -2,9 +2,9 @@
   "name": "tsickle",
   "version": "0.24.1",
   "description": "Transpile TypeScript code to JavaScript with Closure annotations.",
-  "main": "built/src/tsickle.js",
-  "typings": "built/definitions/tsickle.d.ts",
-  "bin": "built/src/main.js",
+  "main": "bazel-bin/src/tsickle.js",
```

Comment:
I'm slightly concerned that the symlinks might do the wrong thing here

you can use
npm pack
 as a dry-run for npm publish to see that it will upload a working package

Reply:
Done.